doc/Administration-guide/Tenants-and-applications-management/index.md

```diff
@@ -10,83 +10,75 @@ sort_idx: 40
+For more information about multitenancy, see [Wikipedia](https://en.wikipedia.org/wiki/Multitenancy).
 
-To create a new application, do the following:
+## Tenant administrator
 
-1. Open the **Applications** window by clicking the corresponding link on the navigation panel.
+Tenant administrator is a Kaa user that manages applications, users, and [event class families]({{root_url}}Programming-guide/Key-platform-features/Events/#event-class-families).
 
-    <img src="attach/image2015-3-4%2016-47-22.png" width="400" height="250">
+To log in to the Administration UI as a tenant administrator, use the previously generated username and password.
+To customize the account, follow the same procedure as described in the [Kaa administrator](#kaa-administrator) section.
 
-2. Click **Add application** at the top of the window.
-3. Enter the title of your application and then click **Add**.
+As a tenant administrator, you can create new applications.
 
-    <img src="attach/image2015-3-4%2016-48-49.png" width="500" height="250">
+To create an application:
 
-**NOTE:** If you open the **Application details** window of the newly created application (by clicking this application on either the **Applications** menu on the navigation panel or the **Applications** window), you will notice that the **Application Token** field has been filled in automatically. This is a unique auto-generated application ID.
+1. Log in to the Administration UI and open the **Applications** page.
+	![](attach/image2015-3-4%2016-47-22.png)
 
-To edit the application, open the **Application details** window by clicking the application name either on the navigation panel or in the list in the **Applications** window.
+2. Click the **Add application** button.
 
-# Tenant developer
+3. On the **Application details** page, enter the title of your application and click **Add**.
+	![](attach/image2015-3-4%2016-48-49.png)
 
-The tenant developer is a user that creates SDKs based on customer requirements. Tenant developers set the Kaa schemas, group endpoints, and control notification processes.
+If you open the **Application details** page of the newly created application, it will display the [application token]({{root_url}}Glossary/#application-token) generated for that application.
 
-To log into the Kaa UI as a tenant developer, use the previously generated credentials for the tenant developer.
+## Tenant developer
 
-To customize this account, click **Settings => Profile** and change the first/last name and e-mail to private ones.
+Tenant developer can create SDKs based on customer requirements, set the Kaa schemas, create [endpoint groups]({{root_url}}Programming-guide/Key-platform-features/Endpoint-groups/), control the [notification]({{root_url}}Programming-guide/Key-platform-features/Notifications/) processes, etc.
 
-To set a private password, click **Settings => Change password**.
+To log in to the Administration UI as a tenant developer, use the previously generated credentials.
```

Comment:
add   
`(or use [default ones]({{root_url}}Getting-started/#administration-ui)) `
at the end of line
because process  of creation is not described on this page